### PR TITLE
Simplify the quickstart steps

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR simplifies the quickstart steps by removing the env addition.
Because these screenshots are made by a not-real quickstart cluster, some places don't add up (for example the piped-id). I'll fix them after releasing.

This PR also renames all application config files to `app.pipecd.yaml` from `.pipe.yaml` because it has been changed by https://github.com/pipe-cd/pipe/pull/2950. It's done with `find` and [renamer](https://github.com/75lb/renamer) like:

```
find . -name .pipe.yaml | renamer --find .pipe.yaml --replace app.pipecd.yaml
```

**Which issue(s) this PR fixes**:

Fixes https://github.com/pipe-cd/pipe/issues/2936

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
Great.
Btw, In order to avoid the broken between those PRs which are related to the translation to `app.pipecd.yaml`, I think we should add the fix for quickstart and docs to this PR too.

**Comment by nakabonne** [comment]
Alright 👍 
/hold

**Comment by nakabonne** [comment]
Because the UI will be changed soon (https://github.com/pipe-cd/pipe/pull/2956), I'm going to update the screenshot as well as sentences around it.
/hold

**Comment by nakabonne** [comment]
Updated the screenshots.

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 1]
```diff
@@ -74,16 +65,24 @@ helm install piped pipecd/piped -n pipecd \
   --set secret.pipedKey.data={YOUR_PIPED_SECRET_KEY}
 ```
 
-### 5. Configuring a kubernetes application
-Navigate to the `Application` page, click on the `Add` button. Then give as:
+### 4. Registering a kubernetes application
+Navigate to the `Application` page, click on the `Add` button on the top left corner.
```
Comment (nghialv) [inline docs/content/en/docs-dev/quickstart/_index.md:69]:
```suggestion
Navigate to the `Applications` page, click on the `Add` button on the top left corner.
```

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 2]
```diff
@@ -74,16 +65,24 @@ helm install piped pipecd/piped -n pipecd \
...
+
+Go to the `ADD FROM GIT` tab, then select:
+- Piped: `dev` (you just registered)
+- CloudProvider: `kubernetes-default`
+
+You should see a lot of application suggestions.
+
+Click the `canary` row. Make sure Kind is set to KUBENETES, and then click the ADD button.
```
Comment (nghialv) [inline docs/content/en/docs-dev/quickstart/_index.md:77]:
```suggestion
You should see a lot of suggested applications.

Select the `canary` application and click the `ADD`button to register.
```

Comment (nakabonne) [reply docs/content/en/docs-dev/quickstart/_index.md:77]: There are lots of apps named canary. I thought the screenshot isn't enough, that's why I added this explanation but do you think it's redundant?

Comment (nghialv) [reply docs/content/en/docs-dev/quickstart/_index.md:77]: By line 73, the user already selected `kubernetes-default` provider so I guess there is only one `canary` application that will be shown, isn't that?

Comment (nakabonne) [reply docs/content/en/docs-dev/quickstart/_index.md:77]: You're right 👍

**Comment by nakabonne** [comment]
@nghialv Thank you! PTAL ;)

File: docs/content/en/docs-dev/quickstart/_index.md [hunk 3]
```diff
@@ -74,16 +65,24 @@ helm install piped pipecd/piped -n pipecd \
...
+### 4. Registering a kubernetes application
+Navigate to the `Applications` page, click on the `Add` button on the top left corner.
+
+Go to the `ADD FROM GIT` tab, then select:
+- Piped: `dev` (you just registered)
+- CloudProvider: `kubernetes-default`
+
+You should see a lot of suggested applications.
+
+Select the `canary` application and click the `ADD`button to register.
```
Comment (nghialv) [inline docs/content/en/docs-dev/quickstart/_index.md:77]:
```suggestion
Select the `canary` application and click the `ADD` button to register.
```

**Comment by nghialv** [comment]
Nice.
Happy to see things are being improved.
/lgtm

**Comment by khanhtc1202** [comment]
Way better than before 🚀 
/approve

**Pull request merged**